modules/local/check_dependencies/main.nf

```diff
@@ -0,0 +1,70 @@
+process CHECK_DEPENDENCIES {
+    tag "${study_id}"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ampcombi:2.0.1--pyhdfd78af_0':
+        'biocontainers/ampcombi:2.0.1--pyhdfd78af_0' }"
+
+    input:
+        tuple val(study_id)
+        tuple path(file_metadata) // Spreadsheet
+        tuple path(analysis_metadata) // Spreadsheet
+        tuple path(workflow_metadata) // Spreadsheet
+        tuple path(read_group_metadata) // Spreadsheet
+        tuple path(experiment_metadata) // Spreadsheet
+        tuple path(specimen_metadata) // Spreadsheet
+        tuple path(sample_metadata) // Spreadsheet
+        tuple path(data_directory) // file path
+    output:
+        path "relational_mapping.json", emit: relational_mapping
+        tuple (path("analysis_types.json")), emit : analysis_types
```

Comment:
WARN: Output `tuple` must define at least two elements

Reply:
Do we also consider to add a status.yml file to record all the errors during the check_dependencies?